lib/access_lint/rails/engine.rb

```diff
@@ -5,7 +5,7 @@ class Engine < ::Rails::Engine
 
       initializer "access_lint-rails" do |app|
         app.config.assets.precompile << "accesslint.js"
-
+        app.config.filter_parameters << :accesslint
```

Comment:
@geoffharcourt maybe, I did this because it add a bunch of noise to the development logs of the host app. Maybe there's a better way.